tests/test_normalizedCalibrationFlux.py

```diff
@@ -0,0 +1,270 @@
+        catalog = self._make_catalog(norm_task.schema)
+
+        ap_corr_map = norm_task.run(catalog=catalog, exposure=self.exposure).ap_corr_map
+
+        self.assertEqual(
+            catalog.schema.getAliasMap().get("slot_CalibFlux"),
+            norm_task.config.normalized_calibflux_name,
+        )
+
+        self.assertIn("base_CompensatedTophatFlux_12_instFlux", ap_corr_map)
+        self.assertIn("base_CompensatedTophatFlux_12_instFluxErr", ap_corr_map)
+
+        # The full set should have a 1.0 ratio when the
+        # aperture flux offset is 0.0
+        ratio = np.mean(catalog["slot_CalibFlux_instFlux"]/catalog[self.ap_name + "_instFlux"])
+        self.assertFloatsAlmostEqual(ratio, 1.0, rtol=1e-10)
+
+        # The subset that was used should always have a 1.0 ratio.
+        used = catalog["apcorr_base_CompensatedTophatFlux_12_used"]
+        ratio_used = np.mean(
+            catalog["slot_CalibFlux_instFlux"][used]/catalog[self.ap_name + "_instFlux"][used]
+        )
+        self.assertFloatsAlmostEqual(ratio_used, 1.0, rtol=1e-10)
+
+        # The error ratios should match the input and output.
+        self.assertFloatsAlmostEqual(
+            catalog["slot_CalibFlux_instFluxErr"]/catalog["slot_CalibFlux_instFlux"],
+            catalog[self.cg_name + "_instFluxErr"]/catalog[self.cg_name + "_instFlux"],
+        )
+
+    def testNormalizedCalibrationFluxOffset(self):
+        np.random.seed(12345)
+        norm_task = self._make_task()
+
+        for offset in [-10.0, 10.0]:
+            catalog = self._make_catalog(norm_task.schema, ap_flux_offset=offset)
+
+            norm_task.run(catalog=catalog, exposure=self.exposure)
+
+            # The full set should not have a 1.0 ratio when the
+            # aperture flux offset is not 0.0
+            ratio = np.mean(catalog["slot_CalibFlux_instFlux"]/catalog[self.ap_name + "_instFlux"])
+            self.assertFloatsNotEqual(ratio, 1.0)
+            # Whether the full set is less than or greater than 1.0 depends on
+            # the sign of the background offset.
+            if offset < 0.0:
+                self.assertGreater(ratio, 1.0)
+            else:
+                self.assertLess(ratio, 1.0)
+
+            # The subset that was used should always have a 1.0 ratio, though
+            # this may be not quite zero because of the trend in the ratio
+            # vs flux even at the bright end.
+            used = catalog["apcorr_base_CompensatedTophatFlux_12_used"]
+            ratio_used = np.median(
+                catalog["slot_CalibFlux_instFlux"][used]/catalog[self.ap_name + "_instFlux"][used]
+            )
+            self.assertFloatsAlmostEqual(ratio_used, 1.0, rtol=1e-10)
+
+    def testNormalizedCalibrationFluxTooFew(self):
+        np.random.seed(12345)
+        norm_task = self._make_task()
+        catalog = self._make_catalog(norm_task.schema)
+
+        flags = np.ones(len(catalog), dtype=bool)
+        flags[0] = False
```

Comment:
Just one valid star is enough to run?

Reply:
Should be... it's meant to be as robust as possible.